scripts/ci/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -ev
+
+echo "Install azdev into virtual environment"
+python -m venv env
+. env/bin/activate
+pip install -U pip setuptools wheel
+pip install $(find ${BUILD_ARTIFACTSTAGINGDIRECTORY}/pypi -name *.tar.gz)
```

Comment:
I'd prefer that we not embed ADO specific variables into our scripts. Mind pulling this out into an argument? This line would become:

`pip install $(find $1/pypi -name *.tar.gz)` or alternatively, we could do the same pushd to repo root with `pushd` and `${BASH_SOURCE[0]}` that we do elsewhere.

Reply:
Offline conversation with @tjprescott, we agreed to address this with a follow-up issue instead of a change here.

Reply:
Here's the issue: #111